cloudinstall/install.py

```diff
@@ -122,10 +126,8 @@ def do_install(self, install_type):
                     self.MultiInstallNewMaas(
                         self.loop, self.ui, self.config).run()
             else:
-                self.ui.set_openstack_rel("Icehouse (2014.1.3)")
```

Comment:
it looks like @battlemidget  didn't want to call ui.set_openstack_rel when headless==true. if that's the case, maybe we shouldn't move it up above? I'll let him comment…